packages/functional-tests/src/app.ts

```diff
@@ -40,6 +41,7 @@ export default class App {
         'clock-sync-test': (): Test => new ClockSyncTest(this, this.baseUrl),
         'primitives-test': (): Test => new PrimitivesTest(this, this.baseUrl),
         'input-test': (): Test => new InputTest(this, this.baseUrl),
+        'gltf-gen': (): Test => new GltfGen(this, this.baseUrl),

```

Comment:
By convention, test name should end in '-test'. --> 'gltf-gen-test'

Reply:
Your addition of gltf-gen is different then every other test. Let's stick with one convention, for discoverability's sake.

Reply:
Agreed it is redundant. Alternatively, could remove '-test' from all of them.

Reply:
While it is redundant, I personally really like that it's there. I love that I can see on the filename if it's a test file (for example text-test.ts vs text.ts). Similarly regarding callstack/autocomplete/find in files, I don't care too much about the string name, though. So I'd rather we kept the word test on all of them

Reply:
It just bugs me that I have to type the word "test" three times when I write up the URL haha. I'll add it.